changelog/1005.misc.rst

```diff
@@ -0,0 +1 @@
+Update base :attr:`Guild.filesize_limit` to 25MB, matching recent API bump.
```

Comment:
```suggestion
Increase the default :attr:`Guild.filesize_limit` from 8MB to 25MB, matching the recent increase by Discord.
```

not sure about "default", maybe "unboosted" works better.

Reply:
yeah. It's also the limit for boost level 1 guilds, so "unboosted" doesn't quite work, but "default" seems fine.